pkg/gatherers/conditional/common.go

```diff
@@ -4,29 +4,32 @@ import (
 	"fmt"
 )
 
+var (
+	ErrAlertPodNameMissing      = fmt.Errorf("alert is missing 'pod' label")
+	ErrAlertPodNamespaceMissing = fmt.Errorf("alert is missing 'namespace' label")
+	ErrAlertPodContainerMissing = fmt.Errorf("alert is missing 'container' label")
```

Comment:
Perhaps it would be good to include the alert name in the errors. WDYT?

Reply:
Well, it is already logging it with the AlertName at `pkg/gatherers/conditional/gather_containers_logs.go#L68`, and I don't want to inject the AlertName into this function, seems a little bit weird...

So, maybe, I can append the error to the `errs` slice at `pkg/gatherers/conditional/gather_containers_logs.go`, with (or something similar):

```go
errs = append(errs, errors.Wrap(err, params.AlertName))
```

WDYT?

Reply:
OK if it's logged with the alert name then it's OK. Thanks